app/src/main/java/com/teamcatchme/catchme/presentation/addcatchu/InitCatchuFragment.kt

```diff
@@ -1,19 +1,79 @@
 package com.teamcatchme.catchme.presentation.addcatchu
 
 import android.os.Bundle
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ForegroundColorSpan
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.TextView
+import androidx.core.content.ContextCompat.getColor
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
 import com.teamcatchme.catchme.R
 import com.teamcatchme.catchme.base.BindingFragment
 import com.teamcatchme.catchme.databinding.FragmentInitCatchuBinding
 
 class InitCatchuFragment :
     BindingFragment<FragmentInitCatchuBinding>(R.layout.fragment_init_catchu) {
+    private val viewModel by activityViewModels<CatchuViewModel>()
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         return super.onCreateView(inflater, container, savedInstanceState)
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        spanString(binding.txtAddcatchu3Title, 4, 6)
+        initButtonClickListener()
+        initObserver()
+    }
+
+    private fun initObserver() {
+        viewModel.catchuImageResId.observe(viewLifecycleOwner) {
+            binding.imgAddcatchu3Catchu.setImageResource(it)
+        }
+        viewModel.catchuName.observe(viewLifecycleOwner) {
+            binding.txtAddcatchu3Name.text = it
+        }
+        viewModel.isCatchuPrivate.observe(viewLifecycleOwner) {
+            when (it) {
+                true -> { // private
+                    binding.btnAddcatchu3Check.setImageResource(R.drawable.ic_checkbox_inactive)
+                }
+                false -> { // not private
+                    binding.btnAddcatchu3Check.setImageResource(R.drawable.ic_checkbox_active)
+                }
+            }
```

Comment:
```suggestion
            binding.btnAddcatchu3Check.setImageResource(
                if(it) R.drawable.ic_checkbox_inactive
                else R.drawable.ic_checkbox_active
            )
```

이게 조금 더 깔끔할 것 같다는 생각